com.brave.Brave.yaml

```diff
@@ -0,0 +1,110 @@
+app-id: com.brave.Brave
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: brave
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--socket=wayland'
```

Comment:
Can you remove that? Brave doesn't natively support Wayland.

Reply:
done